pipeline_config.json

```diff
@@ -19,6 +19,10 @@
     "e5b3d475-4e73-41a0-80a2-b39b28aad89c",
     "856a55d8-9159-4970-bd22-ede3ae5f94e0"
   ],
+  "PhoneNumberUuidTable": {
+    "FirebaseCredentialsFileURL": "gs://avf-credentials/avf-id-infrastructure-firebase-adminsdk-6xps8-b9173f2bfd.json",
+    "TableName": "ADSS"
```

Comment:
I think I'd prefer ADSS_phone_no_avf_phone_id

It's possible that we'll want to use the same infrastructure for storing other mappings on the same project, so probably worth being a bit extra-explicit.

Reply:
changed